packages/jaeger-ui/src/types/tracking.tsx

```diff
@@ -22,7 +22,7 @@ export interface IWebAnalyticsFunc {
 
 export default interface IWebAnalytics {
   init: () => void;
-  context: boolean | RavenStatic | null;
+  context: boolean | typeof BrowserClient | null;
```

Comment:
This field is rather confusing to me. Is it a readable property for the user of IWebAnalytics interface? Or is it an input to the implementation of the interface? If it's the latter, why does it need to be exposed in the interface?

Not specifically related to the PR, but I am trying to understand the structure of the code.

Reply:
So here, `IWebAnalytics` is an interface of the object that is returned by the function `IWebAnalyticsFunc` that takes in 3 variables as the input.

This is used in `ga.tsx` as function `GA`. There, if you notice, context is assigned to `BrowserClient`, and not an instance of it. That is why I have kept context as either a boolean, null, or a type of BrowserClient object, and not an instance of it.

Even I am a little confused as to the places it being used.

Reply:
It is set depending on whether error tracking is enabled on not. It needs to be there on the interface because depending on the config, this needs to be set. 

It is used in L56-57 in `packages/jaeger-ui/src/utils/tracking/index.tsx`